### DISCUSSION TITLE
Issue extracting the Source Sans font from the .tar.gz archive

### DISCUSSION THREAD

**Discussion by mattsches** [Laravel-Backpack/community-forum#842] [Q&A (Help)] [answered]
Hi, I've just been migrating an application to v6 and got the following error:

`unable to open new phar "/var/www/html/vendor/backpack/theme-coreuiv2/resources/assets/fonts/source-sans-3.052R.tar" for writing {"view":{"view":"/var/www/html/vendor/backpack/theme-coreuiv2/resources/views/inc/theme_styles.blade.php","data":[]},"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): unable to open new phar \"/var/www/html/vendor/backpack/theme-coreuiv2/resources/assets/fonts/source-sans-3.052R.tar\" for writing at /var/www/html/vendor/backpack/basset/src/Helpers/Unarchiver.php:103)`

I assume that this is a Linux permission issue, because in my setup the web server user is not allowed to write to `/var/www/html/vendor/backpack/theme-coreuiv2/resources/assets/fonts/` - but extracting `source-sans-3.052R.tar.gz` requires the directory to be writable to create `source-sans-3.052R.tar`. Am I right about this?

So my question is - will this happen each time I deploy my application? Or will this step be omitted if the extracted font files do already exist in the storage directory `storage/app/public/basset/source-sans-pro`?

For reasons, it may be difficult to make the dir writable during deploy, so I'd be glad if I could find an alternative solution. Thanks!

**Comment by pxpm** [comment, marked as answer]
Hey @mattsches thanks for the report. Unfortunately it's something we are aware and it's a long overdue issue. https://github.com/Laravel-Backpack/basset/issues/109

It was in the theme repository and was not prioritized accordingly 😞 I've just picked on it, and we have already a PR pending for it. https://github.com/Laravel-Backpack/basset/pull/110

Your report and the solution provided in the mentioned PR gave me clues on how we could probably solve this in our side without you guys having to hack any other solutions. 

I hope we can have the PR merged today if possible, I need @promatik to have a look at it to make sure I am not breaking nothing, he knows basset in's and out's better than me 🙏 

I will close this for now, you can track this issue on the PR or in the bug report. 

Cheers

**Reply by mattsches** [reply]
I've just upgraded `basset` to version `1.2.3` and can confirm the issue is resolved! Thank you so much :smile:
